ethics/32-assembly_vote.md

```diff
@@ -0,0 +1,511 @@
+ce qui pourrait conduire à des risques de désanonymisation des bulletins.
+
+Bon on pourrait... non ça ne nous emmènerait trop loin de détailler ces problèmes...
+Pas aujourd'hui...
+
+
+## Au delà du principe "à chaque électeur son élu"
+
+De façon très générale, ce qu'on attend d'un scrutin,
+c'est un système qui collecte des jugements des différents électeurs,
+et qui combine ces jugements pour sélectionner dans notre cas une assemblée d'objets,
+de sorte que cette assemblée représente les avis des électeurs.
+
+Or jusque là, tous les scrutins qu'on a introduits s'appuient uniquement sur un système de soutien.
+Ce qui est retenu des jugements des différents électeurs est uniquement les premiers choix
+de leurs bulletins.
+
+Bien sûr, cela a l'avantage d'augmenter les chances 
+que chaque électeur puissent identifier des élus qui représentent parfaitement l'électeur.
+Et jusque là, le gros de la recherche des scrutins d'assemblée
+s'appuient sur le principe philosophique selon lequel 
+chaque électeur devrait pouvoir s'identifier avec au moins un élu.
+
+Cependant, au moins pour d'autres applications,
+un électeur pourrait davantage vouloir que l'ensemble des élus forme une assemblée satisfaisante à ses yeux.
+Autrement dit, plutôt que de vouloir avoir un élu extrêmement représentatif de ses préférences,
+l'électeur pourrait vouloir une assemblée qui, dans son ensemble,
+soit relativement proche de ses préférences.
+
+En particulier, en pratique, 
+on a non seulement des avis sur les candidats qui devraient être élus,
+mais aussi des réticenses, voire du dégoût, pour d'autres candidats.
+
+Typiquement, j'ai vraiment envie que Tournesol tienne compte du fait 
+que, selon moi, certaines vidéos ne devraient absolument pas être recommandées ;
+comme l'ethics-washing des Big Techs, les appels au meurtre et la pédophilie.
+
+De la même manière, au-delà des premiers choix,
+il y a certainement des représentants politiques qui, selon vous,
+ne devraient absolument représenter la France au Parlement européen.
+Est-ce qu'il ne serait pas judicieux de tenir compte aussi des avis négatifs ?
+Un candidat détesté par 90% de la population devrait-il être élu ?
+Devrait-il davantage être élu qu'une candidate que 90% de la population trouve "correct",
+mais que personne ne soutient comme étant son représentant idéal ?
+
+Une vidéo qu'une moitié de la population veut absolument recommander,
+mais qu'une autre moitié ne veut absolument pas recommander,
+doit-elle être préférée à une vidéo qui est unanimement jugée "ok" ?
+
+Identifier et mettre en avant les contenus consensuellement bons, plutôt que ceux qui divisent,
+c'est ce que proposent notamment les Community Notes des Twitter,
+pour sélectionner les notes contextuelles qui seront affichées sur le réseau social.  
+https://communitynotes.twitter.com/guide/en/under-the-hood/ranking-notes#Matrix_Factorization
+
+C'est aussi ce que permet la plateforme délibérative Pol.is,
```

Comment:
Attention à ne pas prononcer pol-point-is, qui est l'URL du projet, mais bien Polis qui est le nom propre du système.